.github/renovate.json

```diff
@@ -11,11 +11,42 @@
       "groupName": "npm dependencies",
       "matchDatasources": [
         "npm"
+      ],
+      "minimumReleaseAge": "14 days"
+    },
+    {
+      "groupName": "golang",
+      "matchDatasources": ["docker", "go-version"],
+      "matchPackagePatterns": ["golang"],
+      "minimumReleaseAge": "14 days"
+    },
+    {
+      // Update only patchlevels of major dependencies like kubernetes, controller-runtime and istio.
+      // Minor and major upgrades most likely require manual adaptations of the code.
+      "matchDatasources": ["go"],
+      // https://docs.renovatebot.com/configuration-options/#matchupdatetypes
+      "matchUpdateTypes": ["major", "minor"],
+      "matchPackagePatterns": [
+        "k8s\\.io\/.+",
```

Comment:
There is [group:kubernetes](https://docs.renovatebot.com/presets-group/#groupkubernetes) which could be used to match kubernetes deps

Reply:
replaced the matchPackagePatterns  for `"k8s\\.io\/.+",` with the group